website/docs/configuration/changelog.md

```diff
@@ -24,7 +24,7 @@ postprocessors = [{ pattern = "foo", replace = "bar"}]
 
 ### header
 
-Header text that will be added to the beginning of the changelog.
+Footer template that will be rendered and added to the beginning of the changelog.
```

Comment:
```suggestion
Header template that will be rendered and added to the beginning of the changelog.
```

Reply:
Also, do you think this documentation change will be enough? I'm thinking if we should add another sentence saying that this works similar to body and footer as well. Or maybe a little example?